Simplenote/src/androidTest/java/com/automattic/simplenote/models/NoteTaggerTest.java

```diff
@@ -1,29 +1,31 @@
 package com.automattic.simplenote.models;
 
-import junit.framework.TestCase;
-
-import com.simperium.client.Bucket;
-
-public class NoteTaggerTest extends TestCase {
-
-    public void testURLEncodeTagKey()
-    throws Exception {
-        Bucket<Note> notesBucket = MockBucket.buildBucket(new Note.Schema());
-        Bucket<Tag> tagsBucket = MockBucket.buildBucket(new Tag.Schema());
-
-        Note note = notesBucket.newObject("hola-mundo");
-        note.setTagString("normal spécial");
-
-        NoteTagger tagger = new NoteTagger(tagsBucket);
-        tagger.onSaveObject(notesBucket, note);
-
-        // there should be two tags now
-        assertEquals(2, tagsBucket.count());
-
-        // spécial should be url encoded to sp%C3%A9cial
-        Tag special = tagsBucket.getObject("sp%C3%A9cial");
-        assertNotNull(special);
-        assertEquals("spécial", special.getName());
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class NoteTaggerTest {
+
+    @Test
+    public void testURLEncodeTagKey() {
+//        Bucket<Note> notesBucket = MockBucket.buildBucket(new Note.Schema());
```

Comment:
What is MockBucket and how can I replicate that?

Reply:
I believe mock bucket is part of the Simperium package:

https://github.com/Simperium/simperium-android/blob/develop/Simperium/src/support/java/com/simperium/test/MockBucket.java

Reply:
I'm not able to import the class. Are we missing some dependency?

Reply:
It was in the [`simperium-test-util`](https://search.maven.org/#search%7Cga%7C1%7Csimperium) package but that dependency stopped being published (I'm not sure why), so unfortunately I don't think it will be as simple as adding that dependency.

It might be worth creating a separate project for the simperium testing utils.

Reply:
I will work on that in the next pr. For this pr, I will just remove the class.

Reply:
I will try mocking it with Espresso and see how that goes